com.microsoft.mrtk.spatialmanipulation/ObjectManipulator/ObjectManipulator.cs

```diff
@@ -708,27 +727,7 @@ private void ApplyTargetPose(MixedRealityTransform targetPose)
         /// <param/>
         protected virtual void ModifyTargetPose(ref MixedRealityTransform targetPose, ref TransformFlags modifiedTransformFlags)
```

Comment:
Given that this function is protected and has had it's behavior change from before, we probably need to refactor/rename it to account for the fact that constraints are now interleaved in a previous step rather than applied all at once in this step.

Reply:
Working on a refactor...

Reply:
Confirming we do need interleaving. PR should be good to go.